monkestation/code/modules/library/skill_learning/job_skillchips/shaft_miner.dm

```diff
@@ -0,0 +1,186 @@
+	/obj/item/survivalcapsule/ = list("Survival capsule!", "Capsule! Pop it when the storms come!", "Just like my cabin on the rig!", "Capsule here!", "Ready for the storms!"),
+	/obj/item/style_meter/ = list("Style meter!", "Now I'm stylish!", "Lookin' cool!", "Only way to mine!", "Time to beat my high score!"),
+	/obj/item/stack/spacecash/ = list("Loads of money!", "Money, money, money~", "It's raining money!", "Dosh!", "Dosh! Grab it while you can, lads!", "Cash here!", "My name is loadsamoney!", "Wallets, watch out for the ladies!", "Right, whop it out!", "Money makes the world go 'round!", "All this mining's making me rich!", "Sorry mate, don't take cheques! Just loads of money!"), //Yes, you get a Killing Floor reference too, as a treat.
+
+	//Items: Misc
+	/obj/item/toy/plush/moth/ = list("Bug!", "Found a bug!", "There's a bug here!", "Moth!", "Moth here!", "🐛"),
+
+	//Machines
+	/obj/machinery/mineral/ore_redemption/ = list("ORM! Drop off your rocks and stones!", "It's a mystery to me how all these minerals fit inside!", "Making a deposit!", "Molly!", "Bloody scientists never upgrade this thing..."),
+	/obj/machinery/computer/shuttle/ = list("Shuttle console!", "Away we go!"),
+
+	//Static Objects: Icemoon
+	/obj/structure/flora/ash/chilly/ = list("Got some fruit here!", "Ice pepper plant!", "Cold as ice!", "Frosty peppers!", "Time for some chilly chili!"),
+
+	//Static Objects: Oshan
+	//None yet, but I am leaving it open.
+
+	//Static Objects: Lavaland & Other
+	/obj/structure/flora/rock/ = list("Rock! Or stone?", "Rock!", "Pioneers used to ride these babies for miles."),
+	/obj/structure/geyser/ = list("Chemical geyser!", "Geyser here! Get the chemists!", "Found a geyser!", "Got some mystery chems here!", "Geyser! Probably tastes better than the chef's cooking..."),
+	/obj/structure/flora/ash/cacti/ = list("Cacti here!", "Don't step on this one!", "Yeast Cones?", "Cactus right here!"),
+	/obj/structure/flora/ash/fireblossom/ = list("Fireblossom here!", "Got a fireblossom plant!", "Fireblossom, for when you need to glow!"),
+	/obj/structure/flora/ash/ = list("Mushroom!"), //You just know I had to do this.
+	/turf/closed/mineral/ = list("Dig through here.", "Minerals inside?", "We need to dig here.", "Drill here?")
+))
+
+/obj/item/skillchip/drg_callout
+	name = "D.R.G.R.A.S Skillchip" //Deep Rock Galactic Reactive Alert System (Or ROCK AND STONE)
+	desc = "Smells faintly of alcohol and has an odd coffee stain on it."
+	custom_price = PAYCHECK_CREW
+	complexity = 0
+	skill_name = "Miner Communication"
+	skill_description = "Understand the skills required to rapidly recognize and call out objects you've pointed at to teammates."
+	skill_icon = "bullhorn"
+	cooldown = 5 SECONDS //Honestly, this should be easy to turn off at any time if you don't want it anymore.
+	activate_message = "<span class='notice'>You suddenly understand the need to shout about things you point at.</span>"
```

Comment:
The span here and on the next line can use span_notice as the span_ things are macros.

Reply:
Admittedly I entirely forget those macros exist. I'm so used to this method